pytorch_lightning/accelerators/accelerator.py

```diff
@@ -355,12 +355,8 @@ def clip_gradients(
             model=self.model,
         )
 
-    def on_train_epoch_end(self, outputs: EPOCH_OUTPUT) -> None:
-        """Hook to do something on the end of an training epoch
-
-        Args:
-            outputs: the outputs of the training steps
-        """
+    def on_train_epoch_end(self) -> None:
```

Comment:
```suggestion
    def on_train_epoch_end(self, *args: Any) -> None:
```
just for compatibility?

Reply:
i'm not sure why its a hook at all on the accelerator because it doesn't seem used anywhere - @justusschock @awaelchli thoughts? are accelerators considered beta where we can make these changes w/o going through a full deprecation cycle?

Reply:
yes

Reply:
@awaelchli yes for what? full deprecation cycle?

Reply:
yes to answer Ananth's last quesetion. Yes accelerator is beta and yes the method is not used anywhere and not called by Lightning so just remove it.